go.mod

```diff
@@ -1,4 +1,4 @@
-go 1.15
+go 1.16
```

Comment:
Wouldn't we target a bigger set of users if we kept 1.15? That way users are free to choose from 1.15 and 1.16.

Reply:
I think 1.15 is fine for now. There wasnt anything major in the release that we are using in the repo that requires the bump.

Reply:
I'd consider using the latest a best practice

Reply:
In some cases yes, but you also get the grunt of bugs and security incidents, if there are any. If you want to merge this, sure. Not worth bike shedding on a small thing.